src/git.js

```diff
@@ -59,7 +59,14 @@ module.exports = function gitFactory(repository = '', origin = 'origin') {
          */
         deleteBranch(branchName){
             return git(repository).push([origin, branchName, '--delete'])
-                .then( () => git(repository).deleteLocalBranch(branchName) );
+                .then( () => git(repository).deleteLocalBranch(branchName) )
+                .then( results => {
+                    if (results.success){
+                        //seems like the content depends on git version
+                        results.branch = branchName;
```

Comment:
Do you have some plan to use this value in a message later?

Reply:
I have no idea, yet. This change (maybe the comment isn't explicit enough) was just intended to ensure this function is working as it was previously using the same format.
By upgrading my version of git, the value of the `branch` property changed to `branchName removed` (the message of the git command instead of the branch name as it was previously). My only intent here was to fix an issue from the git client.
Let's see later if we need it or not